<ui version="4.0" >
 <author>Dane Springmeyer</author>
 <class>Ecotourism</class>
 <widget class="QDialog" name="Ecotourism" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OpenOceanMap - Interview</string>
  </property>
  <widget class="QWidget" native="1" name="button_box" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>261</y>
     <width>460</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pbnSelectEcotourism" >
      <property name="font" >
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text" >
       <string>Draw Shapes by Ecotourism Type</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>pbnSelectEcotourism</zorder>
   <zorder></zorder>
  </widget>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>9</y>
     <width>454</width>
     <height>265</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QGroupBox" name="vessel_info" >
      <property name="font" >
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title" >
       <string>Ecotourism Info</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QWidget" native="1" name="home_port_5" >
         <property name="font" >
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>4</number>
          </property>
          <property name="margin" >
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="employee_text" >
            <property name="minimumSize" >
             <size>
              <width>220</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>220</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font" >
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text" >
             <string>Type of Employee:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox" >
            <property name="maximumSize" >
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text" >
              <string>Captain of Boat</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Guide</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Boat Owner</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Other</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" native="1" name="vessel_length" >
         <property name="font" >
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>4</number>
          </property>
          <property name="margin" >
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="vessel_length_text" >
            <property name="minimumSize" >
             <size>
              <width>220</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>220</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font" >
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text" >
             <string>Vessel length (meters) :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="vessel_length_line" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="minimumSize" >
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font" >
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="contextMenuPolicy" >
             <enum>Qt::PreventContextMenu</enum>
            </property>
            <property name="text" >
             <string>79</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" native="1" name="vessel_length_2" >
         <property name="font" >
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>4</number>
          </property>
          <property name="margin" >
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="vessel_motor_text" >
            <property name="minimumSize" >
             <size>
              <width>220</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>220</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font" >
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text" >
             <string>Vessel motor (horsepower):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="vessel_motor_line" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="minimumSize" >
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font" >
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text" >
             <string>179</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" native="1" name="vessel_length_3" >
         <property name="font" >
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>4</number>
          </property>
          <property name="margin" >
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="haul_capacity_text" >
            <property name="minimumSize" >
             <size>
              <width>220</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>220</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font" >
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text" >
             <string>Haul Capacity (kilograms):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="haul_capacity_line" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="minimumSize" >
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font" >
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text" >
             <string>4000</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" native="1" name="home_port" >
         <property name="font" >
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>4</number>
          </property>
          <property name="margin" >
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="home_port_text" >
            <property name="minimumSize" >
             <size>
              <width>220</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>220</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font" >
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text" >
             <string>Home Port:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="home_port_line" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="minimumSize" >
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font" >
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text" >
             <string>hp1</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" native="1" name="home_port_4" >
         <property name="font" >
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <widget class="QLabel" name="workers_text" >
            <property name="minimumSize" >
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>350</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font" >
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text" >
             <string>How many people  work for the company?</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="workers_line" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="minimumSize" >
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font" >
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text" >
             <string>3</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
